src/common.rs

```diff
@@ -230,7 +230,7 @@ pub(crate) fn config_get(section: &str, key: &str) -> Result<String> {
         }
     };
 
-    Ok(value.clone())
+    Ok(value.to_string())
```

Comment:
looks like this might be a separate matter? (or is this related to the dependency bump?)

Reply:
This is needed for rust-ini bump from 1.12.1 to 1.17.0. In 1.12.1, `value` is supposed to be a `String`, because `ini::ini::Properties` is an alias to [`HashMap<String,String>`](https://docs.rs/rust-ini/0.12.1/ini/ini/type.Properties.html), while in 1.17.0, it is redefined as [a dedicated struct](https://docs.rs/rust-ini/0.17.0/ini/struct.Properties.html#method.get), whose `get` method returns `&str`; so a manual conversion to `String` is necessary.

Reply:
makes sense, thanks!